programs/ssl/ssl_test_lib.c

```diff
@@ -84,6 +92,18 @@ int rng_seed( rng_context_t *rng, int reproducible, const char *pers )
         return( -1 );
     }
 #endif
+#if defined(MBEDTLS_TEST_USE_PSA_CRYPTO_RNG)
```

Comment:
This implication is somewhat coincidental. Furthermore, `MBEDTLS_USE_PSA_CRYPTO` itself is not intrinsically incompatible with reproducible mode: it could be made conditionally compatible based on the PSA RNG being compatible, in which case the first block could be removed but not the second one.

Reply:
Yes it makes sense.